tasks/request_status_for_granule/request_status_for_granule.py

```diff
@@ -151,15 +142,15 @@ def get_file_entries_for_granule_in_job(granule_id: str, async_operation_id: str
     """
     sql = f"""
             SELECT
-                {RECOVERFILE_TABLE_NAME}.{RECOVERFILE_FILENAME_KEY} AS {OUTPUT_FILENAME_KEY},
-                {STATUS_TABLE_NAME}.{STATUS_VALUE_KEY} AS {OUTPUT_STATUS_KEY},
-                {RECOVERFILE_TABLE_NAME}.{RECOVERFILE_ERROR_MESSAGE_KEY} as {OUTPUT_ERROR_MESSAGE_KEY}
+                orca_recoverfile.filename AS {OUTPUT_FILENAME_KEY},
+                orca_status.value AS {OUTPUT_STATUS_KEY},
+                orca_recoverfile.error_message as {OUTPUT_ERROR_MESSAGE_KEY}
             FROM
-                {DATABASE_SCHEMA_NAME}.{RECOVERFILE_TABLE_NAME}
-            INNER JOIN {DATABASE_SCHEMA_NAME}.{STATUS_TABLE_NAME} ON {RECOVERFILE_TABLE_NAME}.{RECOVERFILE_STATUS_ID_KEY}={STATUS_TABLE_NAME}.{STATUS_ID_KEY}
+                orca_recoverfile
+            JOIN orca_status ON orca_recoverfile.status_id=orca_status.id
             WHERE
-                {RECOVERFILE_GRANULE_ID_KEY} = %s AND {RECOVERFILE_JOB_ID_KEY} = %s
-            ORDER BY {RECOVERFILE_LAST_UPDATE_KEY} desc
+                granule_id = %s AND job_id = %s
+            ORDER BY last_update desc
```

Comment:
I would maybe order by filename or status and filename. The thought being is that since this is an array, I could easily look at name of file always being consistent in order or status (error first for example).

Reply:
I was thinking they'd want to see the most recent update, but that could get noisy. Changed.